src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
Defining `data` as `any` it can be not readable the code and it is not a best practice, so add the correct type for the `data` parameters
@ben-at-soul-refiner